README.md

```diff
@@ -9,68 +9,51 @@ Development for a bank/accounting interface for the Flux resource manager. Write
 
 ### Build Requirements
 
-flux-accounting requires the following packages to build:
+flux-accounting requires the following Python packages to build:
 
-| centos8       | ubuntu      | version |
-| ------        | --------    | ------- |
-| python3-devel | python3-dev | >= 3.6  |
-| python3-pip   | python3-pip | 20.0.2  |
-| tox           | tox         | 3.15.0  |
+| Package Name | Version |
+| ------------ | ------- |
+| pandas       | 0.25.3  |
 
 ### Install Instructions
 
-You can install the dependencies required by flux-accounting as well as the package itself to be recognized by Flux's command driver `flux(1)` with `make`.
+##### Building From Source
 
-1. Set your **FLUX_INSTALL_PREFIX** environment variable to point to flux-core's installation target:
-
-```
-$ export FLUX_INSTALL_PREFIX = ~/path/to/flux-core/install/
-```
-
-2. Pull down and install the flux-accounting repo:
-
-```
-$ git clone https://github.com/flux-framework/flux-accounting
-$ cd flux-accounting/
-$ make install
+```console
+./autogen.sh
+PYTHON_VERSION=3.7 ./configure
```

Comment:
Maybe we should make a note of the `PYTHON_VERSION` variable below this code block, but in the canonical example not include it since it isn't required?